index.bs

```diff
@@ -673,6 +676,16 @@ Those coming from other languages should carefully note that despite their names
 not 64.
 </div>
 
+<code>bigint</code> is recommended to be used <em>only</em> in circumstances where a <code>[EnforceRange] long long</code>
+or <code>[EnforceRange] unsigned long long</code> would not work in terms of expressivity: that is, only when the value
+ranges greater than 2<sup>53</sup> or less than -2<sup>53</sup>. In particular, APIs should not use the <code>bigint</code>
```

Comment:
You are using "ranges" as a verb here which makes it a bit hard to understand - could you reword?